### DISCUSSION TITLE
Add option to delay notification or ignore permantently

### DISCUSSION THREAD

**Discussion by johnwalk61** [frenck/spook#436] [Ideas]
I have some templates that get data from an integration. Some of these seem  to trigger Spook after a reboot as the integration has not started yet (for instance Places integration). It would be nice if we can ignore this warning permanently or delay the checks until all integrations are loaded.

Hope this makes sense.

BTW love the integration.

**Comment by JoshC1994** [comment]
Have the same issue, i see these errors after reboot, but they are online by the time i check.  i then need to "ignore" the warnings.  Is there a way to delay this or have spook ignore them on a first pass ect.   i dont want the warning to go completey if the an actual issue. 

Unknown entities used in: Manage INV002 Inverter Mode
Warning · Reported by Spook
Spook has found a ghost in your automations 👻

While floating around, Spook crossed path with the following automation:

[Manage INV002 Inverter Mode](https://bjhwdolqkkh57h7qmc0zconpestbypuv.ui.nabu.casa/config/automation/edit/1733195455228) (automation.manage_inv001_inverter_and_soc)

This automation references the following entities, which are unknown to Home Assistant:

sensor.soc_batt_forecast
To fix this error, [edit the automation](https://bjhwdolqkkh57h7qmc0zconpestbypuv.ui.nabu.casa/config/automation/edit/1733195455228) and remove the use of these non-existing entities.

Spook 👻 Your homie.
